triage.md

```diff
@@ -0,0 +1,68 @@
+## Table of Contents
+- [Scope](#scope)
+- [Schedule](#schedule)
+- [What Is Triaging](#what-is-triaging)
+- [How to Triage](#how-to-triage)
+
+## Scope
+These guidelines serve as a primary document for triaging issues in [opensearch-benchmark](https://github.com/opensearch-project/opensearch-benchmark) and [opensearch-benchmark-workloads](https://github.com/opensearch-project/opensearch-benchmark-workloads) project. For maintainers/contributors, it is a guide to ensure that we address all customer issues and address feature requests in a timely manner.
+
+## Schedule
+Every week a maintainer will be assigned on a rotational basis to conduct triage meeting.
+The meeting duration will be no more than 1 hour every Tuesday @1.30 PM PDT/3.30 PM CDT
+
+## What is Triaging
+Triage is a process of grooming issues with correct labels, assigning them owners if available, ensuring that the issues have all the required information so that they are actionable
+
+## How to Triage
+The steps listed below are not exhaustive and can be updated. Start with [opensearch-benchmark](https://github.com/opensearch-project/opensearch-benchmark) followed by [opensearch-benchmark-workloads](https://github.com/opensearch-project/opensearch-benchmark-workloads) project. For each of these projects we can follow the below steps to triage Issues from different categories. Try to cover at least 1 Issue from each of the below listed category.
```

Comment:
project/projects -> repository/repositories

For each of these projects we can follow the steps below to triage issues from different categories. Try to cover at least one issue from each of the below listed categories.